editions/tw5.com/tiddlers/filters/Filter Operators.tid

```diff
@@ -49,6 +55,6 @@ Most steps process the [[selection of titles|Title Selection]] that are supplied
 <<.operator-rows "[tag[Filter Operators]!tag[Order Operators]!tag[Mathematics Operators]!tag[String Operators]!tag[Tag Operators]tag[Special Operators]sort[]]">>
 </table>
 
-A typical step is written as `[operator[parameter]]`, although not all of the operators need a [[parameter|Filter Parameter]]. 
+A typical step is written as `[operator[operand]]`, although not all of the operators need an [[operand|Filter Operand]]. 
```

Comment:
I believe we always user parameter in the documentation, while the JavaScript code refers to them as operands.

Reply:
It makes the railroad diagrams kind of useless if operands are named as parameter there. That was the reason I changed it. And once parameterised transclusions #6666 are merged we will have a big mess with the term parameter. IMO we should get rid of it, since here it can be nicely replaced by operand, which is the right term from my point of view

Reply:
@pmario I see your concern but changing established terminology is also very problematic for end users. Suggest raising this as an explicit question with @Jermolene

Reply:
I think parameters have never been the right term here. I have never seen a Talk reply that talks about operands as parameters. I think parameters in not specific enough.

Reply:
Ouch! I did not realise that the docs are currently inconsistent on this. I think I'm probably responsible for most of the filter operator docs that use the term operand.

I favour switching to "operand" for the reasons that @pmario gives, but perhaps fixing it globally is outside the scope of this PR, and should be the subject of a separate one.

Reply:
@pmario regardless, it is the established term in our documentation and needs proper consideration and an explicit decision if we are going to change it.

Reply:
@Jermolene do consider that we would also need to rename and redirect this tiddler: https://tiddlywiki.com/#Filter%20Parameter

Reply:
Hi @saqimtiaz I think indeed we would ideally make the change consistently across the docs

Reply:
@saqimtiaz .. I did https://tiddlywiki5-gxbj6e4qm-jermolene.vercel.app/#Filter%20Parameter

Reply:
@Jermolene: Nearly every filter operator describes what we are talking about as "parameter":
![image](https://user-images.githubusercontent.com/1133745/226533851-6a599944-486b-4258-a149-1aec09464542.png)

I think, "parameter" is therefore very ingrained in the English-speaking community. While "operand" may be better, especially for core developers, there is _a lot_ of potential for confusion if you suddenly change terminology. At the very least, the tiddler "Filter Parameters" should get a more verbose explanation, not just a redirect.

Reply:
> I think, "parameter" is therefore very ingrained in the English-speaking community. While "operand" may be better, especially for core developers, there is a lot of potential for confusion if you suddenly change terminology. At the very least, the tiddler "Filter Parameters" should get a more verbose explanation, not just a redirect.

I fear the potential for confusion is already present because we are currently using the terms inconsistently. But if it's significantly easier to standardise on "parameter" rather than "operand", I guess it would make sense to go that way.